src/App.test.tsx

```diff
@@ -1,11 +1,12 @@
 import { render, screen, fireEvent } from '@testing-library/react';
 import App from './App';
 import { testData as data, testDataWithFourQuestions } from './static/data';
+import { translationsProvider } from './locales/helpers';
 
 describe('App component', () => {
```

Comment:
Would be nice to have a test scenario to check the configured locale (e.g. to German).

Reply:
Good idea. I've added a couple of tests now.